scripts/globals/mobskills/stygian_flatus.lua

```diff
@@ -16,7 +16,8 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    skill:setMsg(xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.PARALYSIS, 30, 0, 120))
+    local typeEffect = xi.effect.PARALYSIS
+    skill:setMsg(xi.mobskills.mobStatusEffectMove(mob, target, typeEffect, 30, 0, 120))
```

Comment:
failing to see what logic has changed here. same for the other line.

Reply:
it appears this skill was returning typeEffect which wasn't defined. This change that hookstar is making defines what is attempted to be returned.

Reply:
Just return `xi.effect.PARALYSIS` as well.

Reply:
do you want a grannysmith apple or an ambrosia?
Technically loading it into a variable is more efficient. Since we're accessing the data x2 times it would be quicker to access status.lua only once.

Reply:
Yea, mostly the reasoning here for me is because of how LSB is doing their formatting to save on merge conflicts; I'd generally go for the local variable myself